com.oracle.truffle.r.test.native/packages/altreprffitests/altreprffitests/tests/altrepTests.R

```diff
@@ -0,0 +1,279 @@
+stopifnot(require(altreprffitests))
+
+MAX_NUMBER_OF_ERRORS <- 1
+
+#' Generates an ALTREP instance of vec_wrapper class with just a portion of methods registered.
+#' Note that this portion is randomly chosen.
+generate_random_instance <- function() {
+    generate_random_data <- function() {
+        len <- round( runif(1, min=2, max=30))
```

Comment:
minor: `sample(2:30, 1)` should do the (almost) same and is more idiomatic.

Reply:
..and you can sample items from any vector: `sample(c(T,F), 1)` -> gives `T` or `F`